src/components/Button/Button.js

```diff
@@ -42,6 +42,7 @@ const Button = styled.button`
 
   outline: none;
 
+
   &:hover:not(:disabled),
   &:focus:not(:disabled) {
     background-color: ${get('colors.primary.alternate')};
```

Comment:
Based on discussion with the UX slack channel, this should be changed to `'colors.primary.dark'`.